utils/errors.go

```diff
@@ -16,8 +16,8 @@ func (e ErrorInvalidKey) Error() string {
 
 // ErrorInvalidValue represents an error caused by an invalid value.
 type ErrorInvalidValue struct {
-	Key    string
-	Reason string
+	Key    string `json:"Key"`
```

Comment:
@sebbalex Uppercase ok here?

Reply:
Nope they are not, but we can't break the backward compatibility right now, they are going to be deprecated in next release.
Issue is still open to track this https://github.com/italia/publiccode-validator/issues/1#issuecomment-718094087